internal/storage/local_chunk_manager.go

```diff
@@ -258,3 +261,14 @@ func (lcm *LocalChunkManager) RemoveWithPrefix(prefix string) error {
 	}
 	return lcm.MultiRemove(filePaths)
 }
+
+func (lcm *LocalChunkManager) GetModTime(filename string) (time.Time, error) {
+	absPath := path.Join(lcm.localPath, filename)
+	fi, err := os.Stat(absPath)
+	if err != nil {
+		log.Error("stat fileinfo error", zap.String("filename", filename))
+		return time.Now(), err
```

Comment:
Returning `time.Now()` is confusing and error-prone, just return the default zero value.

Reply:
ok

Reply:
ok